packages/app-project/stores/Recents.js

```diff
@@ -0,0 +1,66 @@
+import { autorun } from 'mobx'
+import { addDisposer, flow, getRoot, types } from 'mobx-state-tree'
+import auth from 'panoptes-client/lib/auth'
+import asyncStates from '@zooniverse/async-states'
+
+export const Recent = types
+  .model('Recent', {
+    subjectId: types.string,
+    locations: types.frozen({})
+  })
+
+const Recents = types
+  .model('Recents', {
+    error: types.maybeNull(types.frozen({})),
+    recents: types.optional(types.array(Recent), []),
```

Comment:
`types.array` defaults to wrap itself with optional:

> Note that since MST v3 types.array and types.map are wrapped in types.optional by default, with [] and {} set as their default values, respectively.

I think they realized that pretty much everyone always does this when defining their models, so it's the default behavior now.

Reply:
Sounds good. I learned the other day that MST also expands `someNumber: 5` to `someNumber: types.optional(types.number, 5)` and ditto (I think) for booleans and strings.